Userland/Libraries/LibGUI/GlyphMapWidget.h

```diff
@@ -92,6 +91,9 @@ class GlyphMapWidget final : public AbstractScrollableWidget {
     virtual void resize_event(ResizeEvent&) override;
     virtual void did_change_font() override;
     virtual void context_menu_event(ContextMenuEvent&) override;
+    virtual void enter_event(Core::Event&) override;
+    virtual void leave_event(Core::Event&) override;
+    virtual void on_automatic_scrolling_timer_fired() override;
```

Comment:
The `on_` prefix is typically used for `Function` hooks.
For callbacks like this, can we call it something with `did` e.g `automatic_scrolling_timer_did_fire`?